parlai/agents/memnn/modules.py

```diff
@@ -61,13 +61,15 @@ def _score(self, output, cands):
             raise RuntimeError('Unexpected candidate dimensions {}'
                                ''.format(cands.dim()))
 
-    def forward(self, xs, mems, cands=None):
+    def forward(self, xs, mems, cands=None, pad_mask=None):
         """One forward step.
 
-        :param xs:    (bsz x seqlen) LongTensor queries to the model
-        :param mems:  (bsz x num_mems x seqlen) LongTensor memories
-        :param cands: (num_cands x seqlen) or (bsz x num_cands x seqlen)
-                      LongTensor with candidates to rank
+        :param xs:       (bsz x seqlen) LongTensor queries to the model
```

Comment:
Go ahead and reformat to the newer format since you're editing this, please

Example:
https://github.com/facebookresearch/ParlAI/blob/1b4f655dbb277a241d332e8fff70308088faaa4b/parlai/core/torch_agent.py#L875-L890